docs/client.rst

```diff
@@ -72,18 +72,111 @@ WBEM Operation recording
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 The WBEM client library API provides the possibility to record the WBEM
-operations that are executed on a connection. This is disabled by default
-and can be enabled by setting the
-:attr:`~pywbem.WBEMConnection.operation_recorder` instance variable of the
-:class:`~pywbem.WBEMConnection` object to an operation recorder object,
-i.e. to an object of a subclass of :class:`~pywbem.BaseOperationRecorder`.
-
-Typical usage scenarios for operation recorders are the tracing of WBEM
-operations, or the generation of test cases.
-
+operations that are executed on a connection.
 WBEM Operation recording uses the classes and subclasses defined in
 :ref:`WBEM Operation Recorder`.
 
+This is disabled by default and can be enabled by adding recorders to a
+WBEMCOnnection with the method
+:method:`~pywbem.WBEMConnection.add_operation_recorder`.
+
+Typical usage scenarios for operation recorders are the tracing of WBEM
+operations, tracing of interactions with a WBEM server, or the generation of
+test cases.
+
+Before pywbem version 0.11.0, there was only a single operation recorder and
+activating/deactivating the recorder was a matter of setting the
+value of an instance of the TestClientRecorder into the WBEMConnection attribute
+:attr:`~pywbem.WBEMConnection.operation_recorder`
+Setting the value of TestClientRecorder into this
+attribute activated and enabled the recorder.  Setting it to None
+deactivated the recorder.
+
+Effective version 0.11.0, pywbem added a second recorder and the requirement
+that both recorders operate simultaneously. If both are active and
+enabled, operation data can be both recorded to a yaml file and to the log
```

Comment:
On "recorded to a yaml file": The link to the previously mentioned TestClientRecorder is missing. It would be good to mention that these are testcases, and/or to mention class TestClientRecorder.

Reply:
Added Mention of  both of the recorder classes here.